### PR TITLE
Separate c from r

### PR DESCRIPTION
**Pull request by PavelBal**

### CODE REVIEW

File: ClrFacade/ClrFacade.cs [hunk 1]
```diff
@@ -196,14 +196,6 @@ private static object[] changeArgumentTypes(object[] arguments, MethodInfo metho
             return result;
         }
 
-        /// <summary>
```
Comment (PavelBal) [inline ClrFacade/ClrFacade.cs:199]: First attempts to remove Mono

File: ClrFacade/ClrFacade.csproj [hunk 2]
```diff
@@ -19,8 +19,9 @@
     <PackageReleaseNotes>Migration to .NET Standard 2.0 and supporting R 3.5.x</PackageReleaseNotes>
     <!-- forces SDK to copy dependencies into build output to make packing easier -->
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <Platforms>AnyCPU</Platforms>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="R.NET" Version="1.8.1" />
```
Comment (PavelBal) [inline ClrFacade/ClrFacade.csproj:24]: Updated R.NET to the latest.

File: ClrFacade/RDotNetDataConverter.cs [hunk 3]
```diff
@@ -32,7 +32,7 @@ private RDotNetDataConverter (string pathToNativeSharedObj)
                 else
                 {
                     dllName = Path.Combine(libDir, Environment.Is64BitProcess ? "x64" : "i386",
```
Comment (PavelBal) [inline ClrFacade/RDotNetDataConverter.cs:34]: 2DO: architecture part of the path sohuld be removed?

File: ClrFacade/app.config [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+	<startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7.2"/></startup></configuration>
```
Comment (PavelBal) [inline ClrFacade/app.config:3]: This is still .NETFramework?

File: rClr.props [hunk 5]
```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ImportGroup Label="PropertySheets" />
+  <PropertyGroup Label="UserMacros">
+    <RInstallPath>C:\PROGRA~1\R\R-43~1.1</RInstallPath>
```
Comment (PavelBal) [inline rClr.props:5]: I had to hard-code the path to R installation here, this should be done automatically somewhow.

**Comment by PavelBal** [comment]
Removed all the R code.

File: 2DO.txt
Comment (Yuri05) [inline 2DO.txt:1]: **2R2D** would be a better name 😆

File: ClrFacade/RDotNetDataConverter.cs (hunk 3 above)
Comment (Yuri05) [reply ClrFacade/RDotNetDataConverter.cs:34]: I think yes, but not 100% sure. let's create an issue for this

File: RclrTests/RclrTests.csproj [hunk 6]
```diff
@@ -17,6 +17,7 @@
     <RepositoryUrl>https://github.com/jmp75/rclr</RepositoryUrl>
     <PackageProjectUrl>https://github.com/jmp75/rclr</PackageProjectUrl>
     <PackageReleaseNotes>Migration to .NET Standard 2.0 and supporting R 3.5.x</PackageReleaseNotes>
```
Comment (Yuri05) [inline RclrTests/RclrTests.csproj:19]: outdated

Comment (PavelBal) [reply RclrTests/RclrTests.csproj:19]: Yeah, there are tons of outdated stuff.. have to remove/update it step by step.

**Pull request merged**